__tests__/phone/validators.test.ts

```diff
@@ -14,3 +14,25 @@ describe('isBrazilianPhone', () => {
     expect(isBrazilianCellPhoneFormat(notPhone as any)).toBeFalsy()
   })
 })
+
+describe('isSamePhone', () => {
+  const samePhones = [
+    { phone1: '47996919816', phone2: '47996919816' },
+    { phone1: '4796919816', phone2: '4796919816' },
+    { phone1: '47996919816', phone2: '4796919816' }
+  ]
+
+  const differentPhones = [
+    { phone1: '47996919816', phone2: '47986919816' },
+    { phone1: '4796919816', phone2: '4786919816' },
+    { phone1: '55996919816', phone2: '47996919816' }
```

Comment:
in my opinion it should be the same phone, is not?

Reply:
No because the DDD

Reply:
can you add a test in the valid same phones, lie:

5547996919817
47996919817

because they should be true

Reply:
Oh, I forgot the DDI, thank youu